src/errors/types.js

```diff
@@ -80,7 +80,6 @@ export const RUNTIME_ERRORS = {
     cannotFindBrowser:                                                'E1005',
     browserProviderNotFound:                                          'E1006',
     browserNotSet:                                                    'E1007',
-    testSourcesNotSet:                                                'E1008',
```

Comment:
I think we should keep this error and emit it if we have no test files even after scanning default directories. But we need to change the message to smthg like:
```
No test files are found with the specified test file patterns or in the default test directories.
```

Could you pls help us, @VasilyStrelyaev?

Reply:
I thought `E1009` is thrown when no files are found:

```
$ testcafe chrome no-such-file.js
ERROR No tests to run. Either the test files contain no tests or the filter function is too restrictive.
```
You want to rephrase `E1009` with this text?

Reply:
> @AndreyBelym I thought `E1009` is thrown when no files are found:
> 
> ```
> $ testcafe chrome no-such-file.js
> ERROR No tests to run. Either the test files contain no tests or the filter function is too restrictive.
> ```
> 
> You want to rephrase `E1009` with this text?

No. We want to change `E1008` message. Now it says that `test sources not set`, that means that we do not calls the `runner.src` method or do not pass sources in cli. Since after this PR sources will be set every time, we want to improve the message and check if the files from the 'src' method or frome the cli, or from default directories, really exist

Reply:
Yes, we should separate these two cases. We broke the `E1008` a long time ago actually, and I always get the `E1009` message when I make a mistake in a test file name. It's really confusing since I start checking the syntax of my file instead of checking the command line parameters.

Reply:
`E1009` is for the situation when you try to start a test with an incorrect syntax.

Reply:
Checked:

```
The specified glob pattern does not match any file or the default test directories are empty.
```